pymc/tests/test_sampling.py

```diff
@@ -1212,3 +1213,54 @@ def test_sample_deterministic():
         idata = pm.sample(chains=1, draws=50, compute_convergence_checks=False)
 
     np.testing.assert_allclose(idata.posterior["y"], idata.posterior["x"] + 100)
+
+
+class TestDraw(SeededTest):
+    def test_draw_one_variable(self):
+        with pm.Model():
+            x = pm.Normal("x")
+
+        x_draws = pm.draw(x)
+        assert x_draws.shape == (1,)
+
+        with pm.Model():
+            x = pm.Normal("x")
+
+        num_draws = 100
+        x_draws = pm.draw(x, draws=num_draws)
+        assert x_draws.shape == (num_draws,)
```

Comment:
```suggestion
```

Actually now that I see, I don't think we need this, we are testing `num_draws` in the test below.

Reply:
Hi yes, It will be removed